Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -28,9 +28,16 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
     public var imageCollectionView: UICollectionView!
     public var numberOfImages: Int = 0
 
+    public var backgroundColor: UIColor = UIColor.blackColor()
+
     public var currentPage: Int {
         set(page) {
-            imageCollectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: page, inSection: 0), atScrollPosition: .CenteredHorizontally, animated: false)
+            if page <= numberOfImages - 1 {
+                imageCollectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: page, inSection: 0), atScrollPosition: .CenteredHorizontally, animated: false)
+            } else {
+                imageCollectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: numberOfImages - 1, inSection: 0), atScrollPosition: .CenteredHorizontally, animated: false)
```

Comment:
You call this method enough times it would make sense to simplify it in a private helper function like

```swift
private fund scrollToImage(withIndex:Int, animated:Bool = false) {
    imageCollectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: withIndex, inSection: 0), atScrollPosition: .CenteredHorizontally, animated: animated)
}
```